eng-wasm/derive/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "eng-wasm-derive"
-version = "0.1.6" # upgrade version in eng-wasm dependancy as well (if needed)
+version = "0.1.7" # upgrade version in eng-wasm dependancy as well (if needed)
```

Comment:
please remove the comment, since it's not relevant anymore

Reply:
done